wpiunits/src/generated/main/java/edu/wpi/first/units/measure/Dimensionless.java

```diff
@@ -277,6 +277,17 @@ default Per<DimensionlessUnit, PowerUnit> divide(Power divisor) {
   }
 
 
+  @Override
+  default Mult<DimensionlessUnit, ResistanceUnit> times(Resistance multiplier) {
+    return (Mult<DimensionlessUnit, ResistanceUnit>) Measure.super.times(multiplier);
+  }
```

Comment:
The multiplication setup for `Dimensionless` will need to be updated to make this return a `Resistance`

Reply:
and done